Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,64 @@
+var botOptions = ["rock", "paper", "scissors"];
+
+var humanScore = 0;
+
+var computerScore = 0;
+
+function humanRock (e) {
+  var humanChoice = $("#rock").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
+  console.log(botChoice);
```

Comment:
Nice to see that you're using `console.log` to check your work here, but if submitting this HW to me is the course's equivalent to taking the code and "putting it live" you might consider removing the `console.log`s as we generally only `console.log` things in live code if there's a warning or error